hassio/src/addon-view/hassio-addon-router.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  HassRouterPage,
+  RouterOptions,
+} from "../../../src/layouts/hass-router-page";
+import { customElement, property } from "lit-element";
+import { HomeAssistant } from "../../../src/types";
+// Don't codesplit the others, because it breaks the UI when pushed to a Pi
+import "./info/hassio-addon-info-tab";
+import "./config/hassio-addon-config-tab";
+import "./log/hassio-addon-log-tab";
+import { HassioAddonDetails } from "../../../src/data/hassio/addon";
+
+@customElement("hassio-addon-router")
+class HassioAddonRouter extends HassRouterPage {
+  @property({ type: Boolean, reflect: true }) public narrow = false;
```

Comment:
Why do we reflect this?

Reply:
Hmm.... I noticed that as well, I can not remember adding it, I just assumed it came with the rebase

Reply:
It did not, removed